src/server_manager/ui_components/outline-server-settings.html

```diff
@@ -176,13 +226,44 @@ <h3>[[localize('settings-server-info')]]</h3>
             <paper-input readonly value="[[serverId]]" label="[[localize('settings-server-id')]]" hidden$="[[!serverId]]" always-float-label maxlength="100"></paper-input>
           </div>
         </div>
+        <!-- Experiments -->
+        <div id="experiments" class="setting card-section" hidden$="[[!shouldShowExperiments]]">
+            <iron-icon class="setting-icon" icon="icons:build"></iron-icon>
+            <div>
+              <h3>[[localize('experiments')]]</h3>
+              <!-- TODO: link to feedback form -->
+              <p inner-h-t-m-l="[[localize('experiments-description', 'openLink', '<a href=#>', 'closeLink', '</a>')]]"></p>
```

Comment:
inner HTML opens the possibility of vulnerabilities.
It's a good practice to label the messages that contain HTML with a -html suffix to clarify things and avoid mistakes. I have the impression we had established that practice in the client. Can we have the same practice here?

Otherwise, how are you going to prevent the use of text message as HTML or HTML as text messages?

Reply:
We need use `innerHTML` because of quotation issues when calling `localize`. We do not prefix HTML [messages in the client](https://github.com/Jigsaw-Code/outline-client/blob/master/resources/master_messages.json). We can start doing so, but affected message IDs would have to be re-translated. If this is somehting we want to do anyway, I'd recommend creating an issue and doing it in a separate PR.

Reply:
I filed [b/148406157](http://b/148406157)